cosmos/scheduler/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine
+RUN apk add gettext
+COPY ./cosmos /bin/
+ENTRYPOINT ["cosmos"]
```

Comment:
Probably need a better Dockerfile (that doesn't depend on a locally built binary)